core/src/main/java/fj/Show.java

```diff
@@ -164,11 +160,7 @@ public void printlnE(final A a) {
    * @return A show instance.
    */
   public static <A> Show<A> showS(final F<A, String> f) {
-    return new Show<A>(new F<A, Stream<Character>>() {
-      public Stream<Character> f(final A a) {
-        return fromString(f.f(a));
-      }
-    });
+    return new Show<A>(a -> fromString(f.f(a)));
```

Comment:
Again, use the static constructor.  In this case `Show.show(F<A, Stream<Character>>)`.